src/Entity/Article.php

```diff
@@ -84,7 +84,9 @@ public function setTitle(string $title): self
 
     public function getContent(): ?string
     {
-        return $this->content;
+        $text = htmlentities($this->content);
+        return preg_replace('!(((f|ht)tp(s)?://)[-a-zA-Zа-яА-Я()0-9@:%_+.~#?&;//=]+)!i', '<a href="$1">$1</a>', $text);
+        //return $this->content;
```

Comment:
This addition was discussed yesterday and resulted in the closure of the PR.
Is there anything that escapes my observation?